libs/expo/betterangels/src/lib/screens/AddEditClient/PersonalInfo/HmisProfiles.tsx

```diff
@@ -35,14 +37,19 @@ export default function HmisProfiles() {
     setValue('hmisProfiles', []);
   };
 
+  const onItemReset = (index: number) => {
+    resetField(`hmisProfiles.${index}.agency`);
+    resetField(`hmisProfiles.${index}.hmisId`);
```

Comment:
i'm seeing strange behavior when clicking a `reset` button. 


1. change ID#
2. click Reset (nothing seems to happen)
3. change ID (lahsa -> some other)
4. ID # now resets (to original value)

Not sure if it's due referencing by `index` here vs using `keyId`.
Or maybe related to not passing `defaultValue` as option, or not setting it?. Not sure.